lib/src/endpoints/me.dart

```diff
@@ -34,12 +34,22 @@ class Me extends EndpointPaging {
     return Player.fromJson(map);
   }
 
-  Future<Iterable<Track>> recentlyPlayed() async {
-    var jsonString = await _api._get('$_path/player/recently-played');
+  Future<Iterable<PlayHistory>> recentlyPlayed(
+      {int limit, DateTime after, DateTime before}) async {
+    if (after != null && before != null) {
+      throw Exception('Cannot specify both after and before.');
+    }
+
+    var jsonString = await _api._get('$_path/player/recently-played?' +
+        _buildQuery({
+          'limit': limit,
+          'after': after?.millisecondsSinceEpoch,
+          'before': before?.millisecondsSinceEpoch
+        }));
     var map = json.decode(jsonString);
 
     var items = map['items'] as Iterable<dynamic>;
```

Comment:
```suggestion
    final map = json.decode(jsonString);
```
Use finals and remove unnecessary typecast.
(I know it was like this before, but it's an opportunity to tidy up those little things)